inc/admin/dashboard/main.php

```diff
@@ -87,18 +87,20 @@ public function init() {
 		add_filter(
 			'neve_about_us_metadata',
 			function () {
+				$product_pages = [];
+				if ( ! defined( 'OTTER_BLOCKS_VERSION' ) ) {
+					$product_pages[] = 'otter-page';
+				}
+
```

Comment:
Just one question, can we not deal with this in the SDK?

Reply:
Ok, I get your point, I think I can move the filtering based on if the product is installed inside the SDK. Will followup with the changes.

Reply:
I simplified the filter here and moved the logic inside the SDK.